tailwind.config.cjs

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable global-require */
+module.exports = {
```

Comment:
Why are you disabling this eslint rule here?

Reply:
Nothing to disable here, everything is working. and I removed that line.